stable_baselines3/common/base_class.py

```diff
@@ -704,7 +708,10 @@ def load(
             get_system_info()
 
         data, params, pytorch_variables = load_from_zip_file(
-            path, device=device, custom_objects=custom_objects, print_system_info=print_system_info
```

Comment:
Was this auto-reformatted? (I don't object to the change just seems unrelated to the rest of the PR).

Reply:
Yeah, I think vscode formatted it, then the linter didn't roll it back as there's some style flexibility. I actually think it's cleaner this way, so if it looks good to you too I would be happy to leave it.